js/fibo-js.js

```diff
@@ -0,0 +1,111 @@
+let outputPlaceHolder = document.getElementById("fiboOutput");
+let largerPlaceHolder = document.getElementById("numLarger");
+let networkError = document.getElementById("sr-only");
+
+let one_place = document.getElementById("first");
+let second_place = document.getElementById("second");
+let third_place = document.getElementById("third");
+
+let checked = document.getElementById("flexCheckDefault");
+
+function fibo(x) { 
+  i = fiboArray.length;
+  if (fiboArray.length>x) {
+      return(fiboArray[i-1]);
+  }
+      fiboArray.push(fiboArray[i-1]+fiboArray[i-2]);
+      fibo(x);
+}
+let fiboArray = [0,1]
+let i;
+
+window.onload = function() {
+  oldResults();
+};
+
+function compare( a, b ) {
+  if ( a.createdDate < b.createdDate ){
+    return -1;
+  }
+  if ( a.createdDate > b.createdDate ){
+    return 1;
+  }
+  return 0;
+}
+
+serverError=false;
+async function urlToData(url) {
+    try {
+      const response = await fetch(url);
+      if (!response.ok) {
+        throw await response.text();
+      }
+      const data = await response.json();
+      serverError = false;
+      return data;
+    }
+    catch(errorText){ 
+      serverError = true;
```

Comment:
Instead of changing the serverErrror variable. You should display the error here in the catch.